kernels/gemmini/Makefile

```diff
@@ -0,0 +1,42 @@
+.DEFAULT_GOAL := all
+
+include ../../runtime/Makefile.rules
+
+TESTS =
+TESTS += tiled_matmul.acc-dialect.x
+
+CFLAGS += -std=gnu11
+CFLAGS += -Wall -Wextra
+
+
+# Override snax-opt rules to avoid linalg-to-library-call pass
+SNAXOPTACCFLAGS = -p insert-accfg-op{accelerator=gemmini},convert-linalg-to-accfg,convert-accfg-to-csr
+
+tiled_matmul.mlir: tiled_matmul.transform.mlir
+	${MLIROPT} --pass-pipeline='builtin.module(test-transform-dialect-interpreter{bind-first-extra-to-ops=linalg.quantized_matmul}, test-transform-dialect-erase-schedule, linalg-generalize-named-ops)' $^ | sed -E 's/iterator_types =/library_call="gemmini", iterator_types =/gm;t' > $@
+
+%.acc-dialect.snax-opt.mlir: %.preprocfinal.mlir
+	$(SNAXOPT) $(SNAXOPTACCFLAGS) -o $@ $<
+
+%.postproc.mlir: %.snax-opt.mlir
+	cat $< | sed 's/arith.maximumf/arith.maxf/g' | sed 's/arith.minimumf/arith.minf/g' > $@
+
+%.ll.mlir: %.postproc.mlir
+	${MLIROPT} --test-lower-to-llvm -o $@ $<
+
+%.ll: %.postproc.mlir
+	${MLIRTRANSLATE} --mlir-to-llvmir -o $@ $<
+
+%.x: %.ll
+	${CC} -x ir $< -c -O3 -o $@ -Wno-override-module --target=riscv64-unknown-elf -mcpu=generic-rv64 -march=rv64gc
+
+allrun: all
```

Comment:
Note that it does not actually run this, otherwise we'd have to install spike with gemmini extensions